### PR TITLE
Add ScyllaDB connector

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Fixes https://github.com/trinodb/trino/issues/10276

## Release notes

(x) Release notes are required, with the following suggested text:

```markdown
# General
* Add ScyllaDB connector. ({issue}`10276`)
```

### CODE REVIEW

**Comment by mkorolyov** [comment]
@ebyhr Hey, thanks for coming back to this so fast.
How could we proceed with it? Which are the next steps? Do you need some help or anything?

**Comment by guy9** [comment]
Interested, following this as well.

**Comment by mkorolyov** [comment]
@ebyhr Hey, what should be done next in order to complete and merge this PR? i possibly can help with smth

**Comment by guy9** [comment]
I'm also happy to help with this.

**Comment by zseta** [comment]
I'd love this to be merged as well! 🙏

**Comment by DanielHe4rt** [comment]
Almost there!!!

File: plugin/trino-scylla/pom.xml [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    <packaging>trino-plugin</packaging>
+    <description>Trino - Scylla Connector</description>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```
Comment (electrum) [inline plugin/trino-scylla/pom.xml:22]: Without a version, I don't think this does anything

File: plugin/trino-scylla/pom.xml [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                <groupId>jakarta.annotation</groupId>
+                <artifactId>jakarta.annotation-api</artifactId>
+            </dependency>
+
+            <dependency>
```
Comment (electrum) [inline plugin/trino-scylla/pom.xml:27]: These shouldn't be needed since we import the `asm-bom` in the root module

**Comment by mkorolyov** [comment]
@ebyhr Hey, could you please come back to this one? Or, as an option, can I help address reviewers' comments? wdyt?

**Comment by guy9** [comment]
ping @ebyhr

**Comment by Tolsi** [comment]
soo? can I help somehow?

**Comment by guy9** [comment]
Hey @ebyhr , is there any reason this was closed? What is missing to have it merged?

**Comment by mkorolyov** [comment]
@ebyhr hey , how can I help to make it merged? I would like to contribute.

**Pull request closed without merging**